Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -1,258 +1,287 @@
-  id: ClothingBackpackERTSecurity
-  name: ERT security backpack
-  description: A spacious backpack with lots of pockets, worn by Security Officers of an Emergency Response Team.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/ertsec.rsi
-
-- type: entity
-  parent: ClothingBackpackERTLeader
-  id: ClothingBackpackERTMedical
-  name: ERT medical backpack
-  description: A spacious backpack with lots of pockets, worn by Medical Officers of an Emergency Response Team.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/ertmed.rsi
-
-- type: entity
-  parent: ClothingBackpackERTLeader
-  id: ClothingBackpackERTEngineer
-  name: ERT engineer backpack
-  description: A spacious backpack with lots of pockets, worn by Engineers of an Emergency Response Team.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/erteng.rsi
-
-- type: entity
-  parent: ClothingBackpackERTLeader
-  id: ClothingBackpackERTJanitor
-  name: ERT janitor backpack
-  description: A spacious backpack with lots of pockets, worn by Janitors of an Emergency Response Team.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/ertjanitor.rsi
-
-- type: entity
-  parent: ClothingBackpackERTLeader
-  id: ClothingBackpackERTClown
-  name: ERT clown backpack
-  description: A spacious backpack with lots of pockets, worn by Clowns of an Emergency Response Team.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/ertclown.rsi
-
-#Special
-- type: entity
-  parent: ClothingBackpack
-  id: ClothingBackpackHolding
-  name: bag of holding
-  description: A backpack that opens into a localized pocket of bluespace.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/holding.rsi
-    state: holding
-    layers:
-    - state: holding
-    - state: holding-unlit
-      shader: unshaded
-  - type: Clothing
-    equippedPrefix: holding
-  - type: Storage
-    capacity: 9999
-
-- type: entity
-  parent: ClothingBackpackClown
-  id: ClothingBackpackCluwne
-  name: jiggles von jonkerton
-  suffix: Unremoveable
-  description: It's a backpack made by Jonk! Co.
-  components:
-  - type: Sprite
-    sprite: Clothing/Back/Backpacks/cluwne.rsi
-  - type: Unremoveable
-    deleteOnDrop: false
-
+- type: entity
+  parent: Clothing
+  id: ClothingBackpack
+  name: backpack
+  description: You wear this on your back and put items into it.
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Backpacks/backpack.rsi
+    state: icon
+  - type: Item
+    size: 9999
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - back
+  - type: Storage
+    capacity: 100
+  - type: ContainerContainer
+    containers:
+      storagebase: !type:Container
+        ents: []
+  - type: UserInterface
+    interfaces:
+    - key: enum.StorageUiKey.Key
+      type: StorageBoundUserInterface
+  # to prevent bag open/honk spam
+  - type: UseDelay
+    delay: 0.5
+
+- type: entity
+  parent: Clothing

```

Comment:
make this parent off ClothingBackpack so you can delete all the unnecessary lines of code